e2e/etcd_test.go

```diff
@@ -433,9 +433,9 @@ func (ep *etcdProcess) Stop() error {
 }
 
 func (ep *etcdProcess) waitReady() error {
-	readyStrs := []string{"enabled capabilities for version", "published"}
+	readyStrs := []string{"starting server...", "published"}
```

Comment:
this line is important; otherwise the processes aren't guaranteed to negotiate up to the 3.0 api by the time the test runs